shared/src/business/entities/EntityConstants.ts

```diff
@@ -1541,6 +1559,9 @@ export const MAX_ELASTICSEARCH_PAGINATION = 10000;
 export const MAX_SEARCH_CLIENT_RESULTS = 200;
 export const MAX_SEARCH_RESULTS = 100;
 
+export const TEMP_JUDGE_ID_TO_REPRESENT_ALL_JUDGES_SELECTION =
```

Comment:
What is this variable used for?

Reply:
This variable was created to keep the type of `judge user id` consistently a string. The `All Judges` selection, from the drop down, does not technically have an id associated to it, but  judge id is needed for mapping the correct trial sessions to the selected judge.